index.js

```diff
@@ -8,7 +8,8 @@ const
     isError = require('lodash.iserror'),
     stringify = require('./lib/utils').stringify,
     debug = require('./lib/utils').debug,
-    defaultFlushTimeoutMs = 10000;
+    defaultFlushTimeoutMs = 10_000,
```

Comment:
Also, do we need a PR in the main repo to use this new version or will it just work automatically in the next `pnpm i`?

Reply:
we'd need to tag